invenio_accounts/manage.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Perform account operations."""
+
+from __future__ import print_function
+
+from sys import stderr
+
+from invenio.ext.script import Manager
+from invenio.ext.sqlalchemy import db
+
+from .helpers import send_account_activation_email
+from .models import User
+from .validators import validate_email, validate_email_is_available,\
+    validate_nickname, validate_nickname_is_available, validate_password
+
+manager = Manager(usage=__doc__)
+
+
+@manager.command
+@manager.option('--nickname', '-n', dest='nickname')
+@manager.option('--email', '-e', dest='email')
+@manager.option('--password', '-p', dest='password')
+def create(nickname, email, password):
```

Comment:
Can we add a flag option here ``--activate``, which will cause the account to be activated immediately and no notification send to the user.